Server/Api/appsettings.Development.json

```diff
@@ -7,6 +7,6 @@
     }
   },
   "AppSettings": {
-    "ConnectionString": "Server=.\\SQLExpress;Database=WebApiCaseStudy;User Id=sa;Password=1234;MultipleActiveResultSets=True;App=EntityFramework"
```

Comment:
You don't need to send appsettings.development.json file changes. we have to add that file into gitignore